Assets/BU/QuizExample/Scripts/UI/QuizFlowUI/InstructorUI/StateInstructorPreResult.cs

```diff
@@ -80,7 +91,18 @@ private void Start()
 
         public override void OnCustomDataReceive(byte[] data)
         {
-            Debug.Log($"NPA-data:{data}");
+            Debug.Log($"NPA-data:{data[0]}");
+            // RRTT Additional
+            bossReference = GameObject.FindGameObjectWithTag("BossReference");
+            bossList = bossReference.GetComponent<BossList>();
+            GameObject boss = Instantiate(bossList.bossPrefabs[data[0]].gameObject, bossPosition);
+            boss.transform.localScale = new Vector3(8.35f, 8.35f, 8.35f);
```

Comment:
อันนี้เป็น suggestion เฉยๆ นะ จะแก้รึเปล่าก็ได้ 
ตรง scale ถ้าเก็บเป็น variable ไว้ จะยืดหยุ่นกว่าถ้ามีที่อยากเปลี่ยนหลายจุด
```suggestion
            float scaleSize = 8.35f;
            boss.transform.localScale = new Vector3(scaleSize , scaleSize , scaleSize);
```